templates/RN/Projects/AddInitialProjectStructure/.template.config/template.json

```diff
@@ -25,29 +25,19 @@
     "wts.projectName": {
       "type": "parameter",
       "dataType": "string",
-      "replaces": "Param_ProjectName"
+      "replaces": "Param_ProjectName",
+      "fileRename": "Param_ProjectName"
     },
-    "wts.wizardVersion": {
-      "type": "parameter",
-      "dataType": "string",
-      "replaces": "Param_Wizard.Version"
-    },
-    "wts.templatesVersion": {
-      "type": "parameter",
-      "dataType": "string",
-      "replaces": "Param_Templates.Version"
-    },
-    "wts.generationProjectType": {
-      "type": "parameter",
-      "replaces": "Param_ProjectType"
-    },
-    "wts.generationFramework": {
-      "type": "parameter",
-      "replaces": "Param_Framework"
-    },
-    "wts.generationPlatform": {
-      "type": "parameter",
-      "replaces": "Param_Platform"
+    "wts.ProjectNameToLower":
+    {
+      "type": "generated",
+      "generator": "casing",
+      "parameters": {
+        "source":"wts.projectName",
+        "toLower": true
+      },
+      "replaces": "Param_ProjectToLower",
```

Comment:
This doesn´t seem to work on all scenarios. Check folders in: android/src/java/debug/and android/src/java/main/......

Reply:
used "wts.sourceName.casing.lower" instead.
For reference: https://github.com/microsoft/CoreTemplateStudio/issues/243